integration_setup.sh

```diff
@@ -53,6 +58,26 @@ then
 	exit 1
 fi
 
+# we can set whether we want the integration with project folder or without it using environment variable 'SET_PROJECT_FOLDER_SETUP'
+if [[ ${SET_PROJECT_FOLDER_SETUP} == true ]]
+then
+	# make an api request to get the status of the project folder setup
+	project_folder_setup_response=$(curl -s -X GET -u${NC_ADMIN_USERNAME}:${NC_ADMIN_PASSWORD} \
+								${NC_INTEGRATION_BASE_URL}/project-folder-status \
```

Comment:
should we also add `-k` in all of the curl requests? I have open project setup in HTTPS and I cannot use this without `-k` option
```suggestion
	project_folder_setup_response=$(curl -k -s -X GET -u${NC_ADMIN_USERNAME}:${NC_ADMIN_PASSWORD} \
								${NC_INTEGRATION_BASE_URL}/project-folder-status \
```